cypress/support/cypress-support/src/utils.js

```diff
@@ -1288,6 +1288,33 @@ function determineActionFromFeatureFile(testRunnable) {
   return 'CORE';
 }
 
+/**
+ * Builds intent environment and resolves intent using Cypress config module.
+ * @param {string} appId - The durable app ID.
+ * @param {string} programType - The type of program (e.g., 'movie', 'episode').
+ * @returns {Chainable<any>} Cypress chainable for resolved intent.
+ */
+function buildAndResolveIntent(appId, programType) {
+  const runtimeEnv = Cypress.env();
+  const intentEnv = {
+    platform: 'xumo',
+    satClient: runtimeEnv.satClientId,
+    satSecret: runtimeEnv.satClientSecret,
+    queryString: false,
+    contentCatalogTypes: 'vod',
+    customFilters: {
+      durableAppId: appId,
+      vodShowings: 'all',
+      numShowings: 'all',
+    },
+    knownEntities: {},
+    app_metadata: {},
+    runtime: {},
+  };
+
+  return cy.callConfigModule('resolveIntent', [programType, intentEnv]);
```

Comment:
We should be able to just add this call where buildAndResolveIntent is called. The building of the object is unnecessary. The only thing that needs to b e passed to the configMod is appId - everything else can be directly retrieved from cypress.env in configMod.